media/js/src/filters/Filter.jsx

```diff
@@ -353,7 +353,7 @@ export default class Filter extends React.Component {
                             </div>
                         </div>
                         <div className="form-group col-md-2">
-                            <label htmlFor="owner-filter">Owner</label>
+                            <label htmlFor="react-select-2-input">Owner</label>
```

Comment:
Do you need to change the Select id's here too, below this?

Reply:
Hi @nikolas! Um, I mean I don't think so? The IDs that are in the Select components are used for the encompassing DIV on the dom. And then nested several divs in is an Input tag which is where I am getting the IDs that the Label should reference (for accessibility purposes). I think the Select ids can stay as they are if Im not mistaken.

Reply:
Ah okay - i guess react-select just isn't behaving like i'd expect it to.